### ISSUE TITLE
Peer disconnected {'host': '127.0.0.1', 'port': 8447}

### ISSUE THREAD

**Issue by starchik85** [Chia-Network/chia-blockchain#3519]
In log this error, how fix it?
2021-05-03T15:00:31.416 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:34.421 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:37.443 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:40.448 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:43.459 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:46.462 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:49.481 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:52.493 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:55.503 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}
2021-05-03T15:00:58.512 harvester chia.harvester.harvester: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8447}

**Comment by igorcarrasco** [comment]
2021-05-08T03:18:47.422 harvester harvester_server        : WARNING  Peer surpassed rate limit 127.0.0.1, message: new_signage_point_harvester, port 8447 but not disconnecting
2021-05-08T03:18:47.427 farmer farmer_server              : WARNING  Peer surpassed rate limit 127.0.0.1, message: farming_info, port 47404 but not disconnecting
2021-05-08T03:18:47.915 harvester harvester_server        : WARNING  Peer surpassed rate limit 127.0.0.1, message: new_signage_point_harvester, port 8447 but not disconnecting
2021-05-08T03:18:47.921 farmer farmer_server              : WARNING  Peer surpassed rate limit 127.0.0.1, message: farming_info, port 47404 but not disconnecting
2021-05-08T03:18:51.073 harvester harvester_server        : WARNING  Peer surpassed rate limit 127.0.0.1, message: new_signage_point_harvester, port 8447 but not disconnecting
2021-05-08T03:18:51.078 farmer farmer_server              : WARNING  Peer surpassed rate limit 127.0.0.1, message: farming_info, port 47404 but not disconnecting
2021-05-08T03:18:59.004 harvester harvester_server        : WARNING  Peer surpassed rate limit 127.0.0.1, message: new_signage_point_harvester, port 8447 but not disconnecting
2021-05-08T03:18:59.009 farmer farmer_server              : WARNING  Peer surpassed rate limit 127.0.0.1, message: farming_info, port 47404 but not disconnecting
2021-05-08T03:18:59.402 harvester harvester_server        : WARNING  Peer surpassed rate limit 127.0.0.1, message: new_signage_point_harvester, port 8447 but not disconnecting
2021-05-08T03:18:59.407 farmer farmer_server              : WARNING  Peer surpassed rate limit 127.0.0.1, message: farming_info, port 47404 but not disconnecting

**Comment by igorcarrasco** [comment]
I'm wondering if it's related.

**Comment by Zeesy** [comment]
I'm sitting on this:
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
2021-05-23T14:35:37.559 harvester harvester               : INFO     Closed RPC server
2021-05-23T14:35:37.560 harvester harvester               : INFO     Waiting for service _await_closed callback
2021-05-23T14:35:37.562 harvester harvester               : INFO     Service harvester at port 8448 fully closed
2021-05-23T14:35:37.820 daemon chia.daemon.server         : INFO     process chia_harvester returned 0
2021-05-23T14:35:49.290 harvester harvester               : INFO     Started harvester service on network_id: mainnet
2021-05-23T14:35:49.293 harvester harvester               : INFO     Reconnecting to peer {'host': '192.168.0.10', 'port': 8447}
2021-05-23T14:35:49.333 daemon chia.daemon.server         : INFO     Register service {'service': 'chia_harvester'}
2021-05-23T14:35:49.335 daemon chia.daemon.server         : INFO     registered for service chia_harvester
2021-05-23T14:35:49.336 daemon chia.daemon.server         : INFO     {'success': True}
2021-05-23T14:36:22.564 harvester harvester               : INFO     Reconnecting to peer {'host': '192.168.0.10', 'port': 8447}
2021-05-23T14:36:56.544 harvester harvester               : INFO     Reconnecting to peer {'host': '192.168.0.10', 'port': 8447}
2021-05-23T14:37:30.572 harvester harvester               : INFO     Reconnecting to peer {'host': '192.168.0.10', 'port': 8447}
2021-05-23T14:38:04.552 harvester harvester               : INFO     Reconnecting to peer {'host': '192.168.0.10', 'port': 8447}

It never connects

**Comment by oxygen** [comment]
I'm seeing this continously:

```
2021-05-25T01:22:07.933 farmer chia.farmer.farmer         : INFO     peer disconnected {'host': '127.0.0.1', 'port': 8448}
2021-05-25T01:22:07.934 farmer farmer_server              : INFO     -> harvester_handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:08.952 harvester harvester_server        : INFO     Connected with farmer {'host': '127.0.0.1', 'port': 8447}
2021-05-25T01:22:08.951 farmer farmer_server              : INFO     -> handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:09.964 farmer farmer_server              : INFO     Connection closed: 127.0.0.1, node id: a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:09.966 farmer chia.farmer.farmer         : INFO     peer disconnected {'host': '127.0.0.1', 'port': 8448}
2021-05-25T01:22:09.969 farmer farmer_server              : INFO     -> harvester_handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:12.958 farmer farmer_server              : INFO     -> handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:12.958 harvester harvester_server        : INFO     Connected with farmer {'host': '127.0.0.1', 'port': 8447}
2021-05-25T01:22:12.958 farmer farmer_server              : INFO     Connection closed: 127.0.0.1, node id: a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:12.958 farmer chia.farmer.farmer         : INFO     peer disconnected {'host': '127.0.0.1', 'port': 8448}
2021-05-25T01:22:13.974 farmer farmer_server              : INFO     -> harvester_handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:14.996 farmer farmer_server              : INFO     -> handshake to peer 127.0.0.1 a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:14.997 harvester harvester_server        : INFO     Connected with farmer {'host': '127.0.0.1', 'port': 8447}
2021-05-25T01:22:14.997 farmer farmer_server              : INFO     Connection closed: 127.0.0.1, node id: a88d6620ecbea619f349313a230b7360c07129f29131c2a85dec53301d412074
2021-05-25T01:22:15.999 farmer chia.farmer.farmer         : INFO     peer disconnected {'host': '127.0.0.1', 'port': 8448}
```

Not the ports are not the same, it says connected 8447 then disconnected 8448

**Comment by madrigo64** [comment]
Windows 10 x64 20H2 19042.985
 Chia version 1.2.2
I got today this error, and after that all stuck even if GUI show sync ok, but on fact is not.
I noticed only after 13 hours that I don't have any points on space pool
very strange situation
I supposed that on this error harvester will try to reconnect but on fact it stuck.
And if I restart GUI then all ok.
![Screenshot 2021-07-18 174524](https://user-images.githubusercontent.com/2878771/126107454-d14df17a-ee2c-4b12-bb78-8ccf702105c9.png)
![3](https://user-images.githubusercontent.com/2878771/126107458-481db2aa-6b66-4147-b6c4-e9904bd20572.png)
![5](https://user-images.githubusercontent.com/2878771/126107460-0908c9e7-f6c7-4e23-a0f7-c5f9c03af7e4.png)
